specs/core/1_custody-game.md

```diff
@@ -533,7 +598,10 @@ def process_bit_challenge(state: BeaconState, challenge: CustodyBitChallenge) ->
     assert is_valid_indexed_attestation(state, get_indexed_attestation(state, attestation))
     # Verify attestation is eligible for challenging
     responder = state.validators[challenge.responder_index]
-    assert epoch + responder.max_reveal_lateness <= get_reveal_period(state, challenge.responder_index)
+    assert get_current_epoch(state) <= get_randao_epoch_for_custody_period(
+        get_reveal_period(state, challenge.responder_index, epoch) + 1, 
+        challenge.responder_index
+    ) + responder.max_reveal_lateness 
```

Comment:
The original assertion was obviously broke -- there is an epoch on the LHS and a period on the RHS. I tried to convert it into something that makes sense, but nor sure if this is what we want. The new assertion will allow challenges for one full custody period after the current one plus the max reveal lateness.